css/ampelement.css

```diff
@@ -14,6 +14,530 @@
+  display: block;
+}
+
+.i-amphtml-loader-moving-line {
+  display: block;
+  position: absolute;
+  width: 100%;
+  height: 100% !important;
+  background-color: rgba(151, 151, 151, 0.65);
+  z-index: 2;
+}
+
+@keyframes i-amphtml-loader-line-moving {
+  0% {transform: translateX(-100%);}
+  100% {transform: translateX(100%);}
+}
+
+.i-amphtml-loader-line.amp-active .i-amphtml-loader-moving-line {
+  animation: i-amphtml-loader-line-moving 4s ease infinite;
+}
+
+.i-amphtml-loader {
+  position: absolute;
+  display: block;
+  height: 10px;
+  top: 50%;
+  left: 50%;
+  transform: translateX(-50%) translateY(-50%);
+  transform-origin: 50% 50%;
+  white-space: nowrap;
+}
+
+.i-amphtml-loader.amp-active .i-amphtml-loader-dot {
+  animation: i-amphtml-loader-dots 2s infinite;
+}
+
+.i-amphtml-loader-dot {
+  position: relative;
+  display: inline-block;
+
+  height: 10px;
+  width: 10px;
+  margin: 2px;
+  border-radius: 100%;
+  background-color: rgba(0, 0, 0, .3);
+
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, .2);
+  will-change: transform;
+}
+
+.i-amphtml-loader .i-amphtml-loader-dot:nth-child(1) {
+  animation-delay: 0s;
+}
+
+.i-amphtml-loader .i-amphtml-loader-dot:nth-child(2) {
+  animation-delay: .1s;
+}
+
+.i-amphtml-loader .i-amphtml-loader-dot:nth-child(3) {
+  animation-delay: .2s;
+}
+
+@keyframes i-amphtml-loader-dots {
+  0%, 100% {
+    transform: scale(.7);
+    background-color: rgba(0, 0, 0, .3);
+  }
+
+  50% {
+    transform: scale(.8);
+    background-color: rgba(0, 0, 0, .5);
+  }
+}
+
+
+/**
+ * "overflow" element is an element shown when more content is available but
+ * not currently visible. Typically tapping on this element shows the full
+ * content.
+ */
+.i-amphtml-element > [overflow] {
+  cursor: pointer;
+  /* position:relative is critical to ensure that [overflow] is displayed
+     above an iframe; z-index is not enough. */
+  position: relative;
+  z-index: 2;
+  visibility: hidden;
+}
+
+.i-amphtml-element > [overflow].amp-visible {
+  visibility: visible;
+}
+
+/* Polyfill for IE and any other browser that don't understand templates. */
+template {
+  display: none !important;
+}
+
+/**
+ * Authors can set this class on their html tags to provide `border-box` box-sizing
+ * to every element in their document. Individual elements can override as necessary.
+ */
+.amp-border-box,
+.amp-border-box *,
+.amp-border-box *:before,
+.amp-border-box *:after {
+  box-sizing: border-box;
+}
+
+/* amp-pixel is always non-displayed. */
+amp-pixel {
+  display: none !important;
+}
+
+/**
+ * Analytics & amp-story-auto-ads tags should never be visible. keep them hidden.
+ */
+amp-analytics, amp-story-auto-ads {
+  /* Fixed to make position independent of page other elements. */
+  position: fixed !important;
+  top: 0 !important;
+  width: 1px !important;
+  height: 1px !important;
+  overflow: hidden !important;
+  visibility: hidden;
+}
+
+html.i-amphtml-fie > amp-analytics {
```

Comment:
This seems like a wrong place for me for this selector. I'd expect FIE CSS would have it.

Reply:
we can have a separate FIE CSS file later. 
actually, as this is the only FIE-only rule, i'd like to see if this can be removed in some way.